### ISSUE TITLE
Organização do projeto

### ISSUE THREAD

**Issue by fsoaresjunior** [jholdy/Crud-PDO-POO#5]
Vamos nos comunicar por meio de issues, por aqui temos a oportunidade de nos organizarmos melhor.

1º  Acredito que devemos seguir a [PSR-2](http://www.php-fig.org/psr/psr-2/) para formatação de código
2º O que falta ser implementado ou criado?
3º O que acham de utilizarmos o autoload da [PSR-4](http://www.php-fig.org/psr/psr-4/)?

**Comment by jholdy** [comment]
Gostei do autoload do psr-4 mais precisa do composer, sera que vai ficar mais dificil de entender com o composer? ou é melhor usar o PSR-2, fiquei com duvida tambem.

**Comment by fsoaresjunior** [comment]
@jholdy PSR-2 é para estilo do código, onde colocar as {}, indentação, etc. Já a PSR-4 é para autoload, não sei se ficaria mais complicado, mas ficaria mais fácil o desenvolvimento.

**Comment by jholdy** [comment]
Concordo em segir o padrão PSR-2 e usar o PRS-4 para autoload. @fsoaresjunior Você pode atualizar o estilo do projeto para PSR-2, e o altuload para PSR-4. Pode fazer isso?

**Comment by fsoaresjunior** [comment]
sim, só estou meio sem tempo esses dias, mas assim que estiver livre, eu faço isso.
